qadence_protocols/mitigations/twirl.py

```diff
@@ -1,44 +1,31 @@
 from __future__ import annotations
 
 import itertools
+from collections import Counter
 
 import torch
-from qadence import QuantumCircuit, QuantumModel, block_to_tensor, chain, kron
-from qadence.noise.protocols import Noise
-from qadence.operations import I, X, Z
-from qadence.types import BackendName
+from qadence import QuantumCircuit, QuantumModel, X, chain, kron
+from qadence.blocks.utils import unroll_block_with_scaling
+from qadence.measurements.samples import compute_expectation
 from torch import tensor
 
 
 def twirl_swap(n_qubits: int, twirl: tuple, samples_twirl: dict) -> dict:
```

Comment:
I don't know if it would make sense to put more explicit typing for function arguments in general, as we did in the last qadence PR.

Reply:
Fixed